companion/src/firmwares/boards.h

```diff
@@ -286,6 +286,11 @@ inline bool IS_TARANIS_PLUS(Board::Type board)
   return board == Board::BOARD_TARANIS_X9DP || board == Board::BOARD_TARANIS_X9E;
 }
 
+inline bool IS_TARANIS_X9D_2019(Board::Type board)
```

Comment:
needed?

Reply:
not used yet, but we can identify many radio, but not this one